app/javascript/mastodon/reducers/accounts_counters.js

```diff
@@ -126,6 +126,7 @@ export default function accountsCounters(state = initialState, action) {
   case STATUS_FETCH_SUCCESS:
     return normalizeAccountFromStatus(state, action.status);
   case ACCOUNT_FOLLOW_SUCCESS:
+    if (action.alreadyFollowing) { return state; }
```

Comment:
eslint hates this line. I can make it ternary if you prefer, but that seemed *less* clear, not more.

Reply:
You can make it a function call instead